tools/minicluster/minicluster.go

```diff
@@ -129,6 +131,49 @@ func main() {
 			contract, executionAmount, executionGasLimit, executionGasPrice, executionData, client, admins, delegates, d,
 			retryNum, retryInterval, resetInterval)
 		wg.Wait()
+
+		chains := make([]blockchain.Blockchain, numNodes)
+		stateHeights := make([]uint64, numNodes)
+		bcHeights := make([]uint64, numNodes)
+		idealHeight := make([]uint64, numNodes)
+
+		netTimeout := 0
+		if timeout > 50 {
```

Comment:
where does 50 come from?

Reply:
oh, It is from 60 - 10 = consensus.rolldpos.delay - consensus.rolldpos.proposerInterval. I replace this magic num now.